mitmproxy/io/tnetstring.py

```diff
@@ -178,17 +178,17 @@ def load(file_handle: BinaryIO) -> TSerializable:
     if c != b":":
         raise ValueError("not a tnetstring: missing or invalid length prefix")
 
-    data = file_handle.read(int(data_length))
+    data = memoryview(file_handle.read(int(data_length)))
     data_type = file_handle.read(1)[0]
 
     return parse(data_type, data)
 
 
-def parse(data_type: int, data: bytes) -> TSerializable:
+def parse(data_type: int, data: memoryview) -> TSerializable:
     if data_type == ord(b","):
-        return data
+        return data.tobytes()
     if data_type == ord(b";"):
-        return data.decode("utf8")
+        return data.tobytes().decode("utf8")
```

Comment:
```suggestion
        return str(data, "utf8")
```
(I think this should avoid one copy operation)